web/gds-user-ui/src/components/CertificateReview/TrisaImplementationReview.tsx

```diff
@@ -23,9 +23,10 @@ const TrisaImplementationReview = () => {
   useEffect(() => {
     const validate = async () => {
       try {
-        await trisaImplementationValidationSchema.validate(trisa, {
+        const test = await trisaImplementationValidationSchema.validate(trisa, {
           abortEarly: false
         });
+        console.log('[TrisaImplementationReview] test', test);
```

Comment:
Added this to try to understand after the deployment the validation result of step 4